### PR TITLE
Support Jackson serializer

### PR DESCRIPTION
**Pull request by pawelkaczor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ compared to systemd:
   ```sbt
   libraryDependencies += "com.github.eikek" %% "calev-circe" % "0.5.2"
   ```
+- The *jackson* module defines `CalevModule` for [Jackson](https://github.com/FasterXML/jackson)
```
Comment (pawelkaczor) [inline README.md:78]: @eikek fyi Jackson is the default json serializer in Akka 2.6

File: modules/jackson/src/main/scala/com/github/eikek/calev/jackson/CalEventSerializer.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package com.github.eikek.calev.jackson;
```
Comment (eikek) [inline modules/jackson/src/main/scala/com/github/eikek/calev/jackson/CalEventSerializer.java:1]: Maybe the java files can better go into `src/main/java` source dir? Or other question: why not use Scala? :-) Can't it be done using Scala for some reason?

Comment (pawelkaczor) [reply modules/jackson/src/main/scala/com/github/eikek/calev/jackson/CalEventSerializer.java]: Good point 👍🏻

**Comment by eikek** [comment]
Nice :)

**Pull request merged**